aries-site/src/pages/components/skeleton.mdx

```diff
@@ -15,16 +15,19 @@ import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
   <DashBoardSkeletonExample />
 </Example>
 
+### Guidance
```

Comment:
Wondering if we need this section, feels like it is captured by the introduction already:

<img width="856" alt="Screen Shot 2024-02-15 at 3 48 03 PM" src="https://github.com/grommet/hpe-design-system/assets/12522275/519b6843-2e94-4692-a1e4-4f9f70fc8147">

Reply:
Just for some context on how the pages have been built over time, we haven't quite ironed out the sections that we always want across pages. That being said, Spinner is a bit of an older page so I wouldn't rely on it as the "gold standard" for page structure. Some newer pages are Layer, Card, etc.

I think we realized somewhere along the way that the entire docs site is "guidelines"/"guidance" so having a section called that wasn't super meaningful. I'd still opt for removing this section and just altering the description at the top of the page if we need wording changes.

Reply:
I think as a team we should probably revisit what our "gold standard" is for page structure

Reply:
Yes, agreed, Oliver and I are starting to look at that.